style.css

```diff
@@ -1,3 +1,186 @@
-#root {
-  color: red;
+/* Resetting our page styling */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* Setting font style */
+*,
+*::before,
+*::after {
+  font-family: "Open Sans", sans-serif;
+}
+
+/* Page background */
+html,
+body {
+  min-height: 100%;
+  background: linear-gradient(#bdc3c7, #2c3e50);
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+/* Main heading (h1)styling */
+h1 {
+  text-shadow: 3px 3px 5px black;
+  color: whitesmoke;
+  background-color: cadetblue;
+  border-radius: 5px;
+  width: 350px;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 30px;
+  margin-top: 1.5rem;
+}
+
+/* All search fields i.e: shows-dropDown, episodes-dropDown & search bar and search count*/
+#header {
+  background-color: rgb(62, 75, 80);
+  /* display: flex; */
+  /* flex-wrap: wrap; */
+  /* justify-content: space-around; */
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  gap: 1.5rem;
+  border-radius: 1.5rem;
+  padding: 1.5rem;
+  margin: 1rem;
+}
+
+#showSelection {
+  min-width: 200px;
+  border-radius: 0.5rem;
+  background-color: blanchedalmond;
+  font-size: 1.1rem;
+}
+#episodeSelection {
+  min-width: 200px;
+  border-radius: 0.5rem;
+  background-color: blanchedalmond;
```

Comment:
#showSelection, #episodeSelection and #search all use the same styles so you could combine these into one class to make it easier to manage.

Reply:
I'll update this page once I start level 500 :)